instrumentation/rpc/src/main/java/brave/rpc/RpcResponseParser.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2013-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.rpc;
+
+import brave.Span;
+import brave.SpanCustomizer;
+import brave.Tags;
+import brave.Tracing;
+import brave.handler.FinishedSpanHandler;
+import brave.propagation.TraceContext;
+
+/**
+ * Use this to control the response data recorded for an {@link TraceContext#sampledLocal() sampled
+ * RPC client or server span}.
+ *
+ * <p>Here's an example that adds the "rpc.error_code" even though "error" contains it.
```

Comment:
TODO: change this example as it doesnt' actually change anything as we already set "rpc.error_code"